addons/website/models/website.py

```diff
@@ -342,9 +342,10 @@ def configurator_init(self):
 
     @api.model
     def configurator_recommended_themes(self, industry_id, palette):
-        domain = [('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common'])]
+        domain = [('name', '=like', 'theme%'), ('name', 'not in', ['theme_default', 'theme_common']), ('state', '!=', 'uninstallable')]
         client_themes = request.env['ir.module.module'].search(domain).mapped('name')
-        client_themes_img = dict([(t, http.addons_manifest[t].get('images_preview_theme', {})) for t in client_themes])
+        client_themes_img = dict([(t, http.addons_manifest[t].get('images_preview_theme', {}))
```

Comment:
There is now a linter error, sorry to be picky :)
Would be ideal to install one (flake8/pep8)

![image](https://user-images.githubusercontent.com/30048408/169806128-01a4ff65-6d3b-4c9f-b331-19a71fe2aa1b.png)

```py
    client_themes_img = dict([
        (t, http.addons_manifest[t].get('images_preview_theme', {}))
        for t in client_themes if t in http.addons_manifest
    ])
```
^ will be fine (and more easy to read, win-win)